server/organization/views.py

```diff
@@ -14,3 +16,28 @@ class OrganizationViewSet(viewsets.ModelViewSet):
     queryset = Organization.objects.all()
     serializer_class = OrganizationSerializer
     permission_classes = [IsAuthenticated, UserOrganizationPermission]
+
+
+class ModifyOrganizationInventoryItemsDataUpdate(generics.GenericAPIView):
+    """
+    API endpoint that allow a user to update the timing at which
+    the Inventory Data is refreshed
+    """
+    permission_classes = [IsAuthenticated]
+
+    def post(self, request):
+        data = request.data
+        print(data)
```

Comment:
why are you logging here, is this intentional?

Reply:
No i forgot to remove it, I'll delete it